apps/api/v2/src/modules/stripe/stripe.service.ts

```diff
@@ -0,0 +1,15 @@
+import { AppConfig } from "@/config/type";
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import Stripe from "stripe";
+
+@Injectable()
+export class StripeService {
+  public stripe: Stripe;
+
+  constructor(configService: ConfigService<AppConfig>) {
+    this.stripe = new Stripe(configService.get("stripe.apiKey", { infer: true }) ?? "", {
+      apiVersion: "2020-08-27",
```

Comment:
Why this version?

Reply:
No reason, just the one it auto filled, will set to latest.

Reply:
@keithwillcode this is actually hardcoded by Stripe to this version. Doesn't look like we can differ from it, all other Stripe usages in Web App use this version too, and it's set to default in our Stripe dashboard too.